package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "WaveLog_Gate_by_DJ7NT",
+  "name": "wavelog-gate-by-dj7nt",
```

Comment:
@int2001 

This is something we should take a look at. The "old" name was `WaveLog_Gate_by_DJ7NT` but this one can fail in some environments due to the required npm name pattern. Maybe we can just call it `wavelog-gate` here and place DJ7NT in the `productName`  (`Ln 3`) or in the `description`  (`Ln 4`)?

<img width="293" alt="image" src="https://github.com/user-attachments/assets/e98bda99-b0e8-4e82-b1b2-fb8db7022ca5">